site/committers.html

```diff
@@ -6,120 +6,178 @@ <h2>Committers</h2>
 <table class="table"><thead>
 <tr>
 <th>Name</th>
+<th>Role</th>
 <th>Alias (email is &lt;alias&gt;@apache.org)</th>
+<th>Affiliation</th>
 </tr>
 </thead><tbody>
 <tr>
 <td>Jacques Nadeau</td>
+<td>VP, PMC</td>
 <td>jacques</td>
+<td>Dremio</td>
 </tr>
 <tr>
 <td>Todd Lipcon</td>
+<td>PMC</td>
 <td>todd</td>
+<td>Cloudera</td>
 </tr>
 <tr>
 <td>Ted Dunning</td>
+<td>PMC</td>
 <td>tdunning</td>
+<td>MapR</td>
 </tr>
 <tr>
 <td>Michael Stack</td>
+<td>PMC</td>
 <td>stack</td>
+<td>Cloudera</td>
 </tr>
 <tr>
 <td>P. Taylor Goetz</td>
+<td>PMC</td>
 <td>ptgoetz</td>
+<td>Hortonworks</td>
 </tr>
 <tr>
 <td>Julian Hyde</td>
+<td>PMC</td>
 <td>jhyde</td>
+<td>Hortonworks</td>
 </tr>
 <tr>
 <td>Reynold Xin</td>
+<td>PMC</td>
 <td>rxin</td>
+<td>Databricks</td>
 </tr>
 <tr>
 <td>James Taylor</td>
+<td>PMC</td>
 <td>jamestaylor</td>
+<td>Salesforce</td>
 </tr>
 <tr>
 <td>Julien Le Dem</td>
+<td>PMC</td>
 <td>julien</td>
+<td>WeWork</td>
 </tr>
 <tr>
 <td>Jake Luciani</td>
+<td>PMC</td>
 <td>jake</td>
+<td>DataStax</td>
 </tr>
 <tr>
 <td>Jason Altekruse</td>
+<td>PMC</td>
 <td>json</td>
+<td>Workday</td>
 </tr>
 <tr>
 <td>Alex Levenson</td>
+<td>PMC</td>
 <td>alexlevenson</td>
+<td>Twitter</td>
 </tr>
 <tr>
 <td>Parth Chandra</td>
+<td>PMC</td>
 <td>parthc</td>
+<td>MapR</td>
 </tr>
 <tr>
 <td>Marcel Kornacker</td>
+<td>PMC</td>
 <td>marcel</td>
+<td>Independent</td>
 </tr>
 <tr>
 <td>Steven Phillips</td>
+<td>PMC</td>
 <td>smp</td>
+<td>Dremio</td>
 </tr>
 <tr>
 <td>Hanifi Gunes</td>
+<td>PMC</td>
 <td>hg</td>
+<td></td>
 </tr>
 <tr>
 <td>Abdelhakim Deneche</td>
+<td>PMC</td>
 <td>adeneche</td>
+<td>Dremio</td>
 </tr>
 <tr>
 <td>Wes McKinney</td>
+<td>PMC</td>
 <td>wesm</td>
+<td>Two Sigma</td>
 </tr>
 <tr>
 <td>David Alves</td>
+<td>PMC</td>
 <td>dralves</td>
+<td>Cloudera</td>
 </tr>
 <tr>
 <td>Ippokratis Pandis</td>
+<td>PMC</td>
 <td>ippokratis</td>
+<td>Amazon</td>
 </tr>
 <tr>
 <td>Uwe L. Korn</td>
+<td>PMC</td>
 <td>uwe</td>
+<td>Blue Yonder</td>
```

Comment:
Small legal thing: it should be `Blue Yonder GmbH` (german limited-style companies need to take care that they are declared as a limited company on semi-official things)